src/components/ThumbnailsPanel/ThumbnailsPanel.scss

```diff
@@ -18,4 +18,15 @@
     border-color: $gray87;
     margin: 0px;
   }
+
+  .columnsOfThumbnails{
```

Comment:
Can you add one space before the `{`?

Reply:
I fixed the spacing to match other scss files